platforms/ios/build_framework.py

```diff
@@ -424,7 +435,19 @@ def makeFramework(self, outdir, builddirs):
                     "arm64": "arm64-apple-macos",
                 }
             for d in builddirs:
-                copy_tree(os.path.join(d, "install", "lib", name + ".framework", "Modules"), os.path.join(dstdir, "Modules"))
+                for item in os.listdir(os.path.join(d, "install", "lib", name + ".framework", "Modules")):
+                    copied_src_item = os.path.join(os.path.join(d, "install", "lib", name + ".framework", "Modules"), item)
+                    copied_dest_item = os.path.join(os.path.join(dstdir, "Modules"), item)
+                    if os.path.exists(copied_dest_item):
+                        try:
+                            shutil.rmtree(copied_dest_item)
```

Comment:
We should not cleanup destination directories.
AFAIK, artifacts of different archs are merged in one location.